src/containers/PageConstructor/PageConstructor.tsx

```diff
@@ -50,6 +51,7 @@ export interface PageConstructorProps {
     custom?: CustomConfig;
     renderMenu?: () => React.ReactNode;
     navigation?: NavigationData;
+    branded?: boolean;
```

Comment:
`isBranded`
and add info in readme https://github.com/gravity-ui/page-constructor/blob/main/README.md#L42